File: 프로그래머스/0/181847. 0 떼기/0 떼기.cs
using System;

public class Solution {
    public string solution(string n_str) {
        string answer = "";
        int temp = 0;
        for(int i = 0 ; i < n_str.Length;i++)
        {
            if(n_str[i] == '0')
            {
                temp = i+1;
                continue;
            }
            else if(temp == i)
            {
                answer = n_str.Substring(i,n_str.Length-i);
                break;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181847. 0 떼기/0 떼기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string n_str) {
    string answer = "";
    bool trigger = false;
    for(int i = 0; i < n_str.size(); i++)
    {
        if(!trigger && n_str[i] == '0')
            continue;
        else
            trigger = true;
            
        answer += n_str[i];
    }
    return answer;
}